src/components/transport_manager/src/usb/libusb/usb_handler.cc

```diff
@@ -320,7 +324,10 @@ void UsbHandler::Thread() {
   SDL_TRACE("enter");
   int completed = 0;
   while (!shutdown_requested_) {
-    libusb_handle_events_completed(libusb_context_, &completed);
+    // all zero timeval struct for non-blocking mode
+    struct timeval max_wait_time = {0, 0};
+    libusb_handle_events_timeout_completed(
```

Comment:
@vlantonov.
Looks like we start using use all CPU resources for this kind of pulling.
Looks like `liusb` also recommend to use `poll` - http://libusb.sourceforge.net/api-1.0/mtasync.html